worker/jobs/pull/elife.py

```diff
@@ -12,4 +17,39 @@ def pull_elife(source: dict, sink: str) -> str:
     assert "article" in source, "eLife source must have an article number"
 
     url = "https://elifesciences.org/articles/{article}.xml".format(**source)
-    return pull_http(dict(url=url), sink)
+    folder, article = os.path.split(sink)
+
+    assert "{article}.jats.xml".format(**source) == article
+
+    temp_sink = "{}.temp".format(sink)
+    pull_http(dict(url=url), temp_sink)
```

Comment:
Rather than writing to a temporary file (and later having to clean it up), wouldn't it be better to use `requests.get` and pass the XML directly to `etree`?